conda_smithy/ci_register.py

```diff
@@ -178,48 +178,36 @@ def add_project_to_travis(user, project):
     headers = travis_headers()
     endpoint = 'https://api.travis-ci.org'
 
-    url = '{}/hooks'.format(endpoint)
+    url = '{}/repos/{}/{}'.format(endpoint, user, project)
 
-    found = False
     count = 0
 
-    while not found:
+    while True:
         count += 1
-
         response = requests.get(url, headers=headers)
         try:
-            response.raise_for_status()
             content = response.json()
```

Comment:
So the JSON response is well defined even when it fails. However, the `raise_for_status` would actually raise an error. Not sure what we want to do here.